fluent-fallback/src/pin_cell/mod.rs

```diff
@@ -1,97 +0,0 @@
-#![deny(missing_docs, missing_debug_implementations)]
```

Comment:
Thought: Looks like this will be another vendored crate for Firefox, but it should be direct enough to certify on the next vendoring.

Reply:
My logic is that we're not experts to maintain it in-house, so better to vendor no-dep utility